ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -104,6 +104,8 @@ tests:
   skip_if_only_changed: ^docs/|\.md$|^(?:.*\/)?(?:\.gitignore|DOWNSTREAM_OWNERS|DOWNSTREAM_OWNERS_ALIASES|LICENSE|DCO|CODEOWNERS)|\.golangci\.yaml|\.goreleaser\.yml|\.markdownlint.yaml|^\.github|Tiltfile$
   steps:
     cluster_profile: aws
+    env:
+      TEST_ARGS: --monitor=watch-namespaces
```

Comment:
@kuiwang02 Any reason why we do not do that?
We should not run all those checks for our e2e so that we can avoid flakes and issues that go beyond what we can solve.

Reply:
@camilamacedo86 here is the reason https://github.com/openshift/release/pull/67301#issuecomment-3111650125

Reply:
@tmshort ^